src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+
+    @Test
+    void test1() throws DukeException {
```

Comment:
Perhaps you can consider naming the test cases according to what they are testing